Source/WebCore/page/EventHandler.h

```diff
@@ -38,6 +38,7 @@
 #include "TextGranularity.h"
 #include "Timer.h"
 #include <memory>
+#include <utility>
```

Comment:
is this import needed?

Reply:
I'm importing this for `std::pair`. I reckon `<utility>` can be removed safely because it gets transitively included from elsewhere, but I prefer to include-what-I-use ([why IYWU](https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/WhyIWYU.md)) in headers at least... is this against our preferred style? I can remove it then!